dojson/utils.py

```diff
@@ -13,6 +13,25 @@
 import six
 
 
+class IgnoreKey(Exception):
+
+    """The corresponding key has been ignored."""
+
+    pass
+
+
+def ignore_value(f):
```

Comment:
I see that I haven't add this configuration in my initial design of `filter_values`. Hence I think it can stay. Thanks!